client/dogs/SearchPage.js

```diff
@@ -43,40 +37,35 @@ export default class IndexPage extends React.Component {
     const { query } = ctx;
 
     let results = null;
+    //console.log('QUERY', query);
 
-    if (query.q) {
+    if (query.firstName && query.lastName && query.dogName && query.year) {
```

Comment:
Can this code be removed now that the search results are loaded on the Form page?